### PR TITLE
[VEN-2630] Remove ankrBNB Rewards Distributor

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

File: vips/vip-329/bsctestnet.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const COMPTROLLER_IMPL = "0x2CF0e211c99dFd28892cf80D142aA27a9042Dbf4";
+export const COMPTROLLER_TEMP_IMPL = "0x6F16a377323ab1e34a9a7a06a4E68EbcCeB10A72";
```
Comment (GitGuru7) [inline vips/vip-329/bsctestnet.ts:5]: Can you verify this contract?

File: simulations/vip-329/bsctestnet.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  describe("Pre-VIP behavior", async () => {
+    it("check implementation", async () => {
+      const implementation = await beacon.implementation();
+      expect(implementation).to.be.equal(COMPTROLLER_IMPL);
+    });
+
+    it("check reward distributor exists", async () => {
+      const distributors = await comptrollerContract.getRewardDistributors();
+      expect(distributors.length).to.be.equal(5);
```
Comment (chechu) [inline simulations/vip-329/bsctestnet.ts:27]: We can expect `distributors` includes `REWARDS_DISTRIBUTOR_ankrBNB`

File: simulations/vip-329/bsctestnet.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  describe("Post-VIP behavior", async () => {
+    it("check implementation", async () => {
+      const implementation = await beacon.implementation();
+      expect(implementation).to.be.equal(COMPTROLLER_IMPL);
+    });
+
+    it("check reward distributor removed", async () => {
+      const distributors = await comptrollerContract.getRewardDistributors();
+      expect(distributors.length).to.be.equal(4);
```
Comment (chechu) [inline simulations/vip-329/bsctestnet.ts:41]: We can expect `distributors` doesn't include `REWARDS_DISTRIBUTOR_ankrBNB`

**Comment by chechu** [comment]
VIP executed: https://testnet.bscscan.com/tx/0x166c7f7f02ad2afd8057fa3e882330767d836541e048655d6c9a3578a89530ae

**Pull request merged**